provider/testing/resource.go

```diff
@@ -84,7 +84,7 @@ func TestResource(t *testing.T, resource ResourceTestCase) {
 	}); configErr != nil {
 		t.Fatal("failed to configure provider", configErr)
 	} else if resp != nil && resp.Diagnostics.HasErrors() {
-		t.Fatal("errors while configuring provider", configErr)
+		t.Fatal("errors while configuring provider", resp.Diagnostics.Error())
```

Comment:
should we print only errors in `resp.Diagnostics.Error` function or it is ok to print all the `diags`?